packages/state-manager/src/sagas/files/checkForMissingFiles/checkForMissingFiles.saga.test.ts

```diff
@@ -171,17 +171,143 @@ describe('checkForMissingFilesSaga', () => {
       .run()
   })
 
+  test('download only specific file from many missing files', async () => {
```

Comment:
I wish this name was more descriptive, eg. "resume download of all valid files after restarting the app"